open_spiel/python/algorithms/best_response.py

```diff
@@ -115,7 +118,13 @@ def decision_nodes(self, parent_state):
       if parent_state.current_player() == self._player_id:
         yield (parent_state, 1.0)
       for action, p_action in self.transitions(parent_state):
-        for state, p_state in self.decision_nodes(parent_state.child(action)):
+        if parent_state.is_simultaneous_node():
+          yield (parent_state, 1.0)
```

Comment:
This seems to imply yielding the parent state multiple times - is that intended?